### DISCUSSION TITLE
Swedish translation added

### DISCUSSION THREAD

**Discussion by sparmark** [Ruben2776/PicView#203] [General]
I have attached the Swedish translation in sv.json.

Sorry for delivering this way, but I am not familiar with pull requests yet. I hope you can add the file properly.

[sv.json](https://github.com/user-attachments/files/17527525/sv.json)

**Comment by Ruben2776** [comment]
Thank you very much. 

I have added it in now.

I have noticed that it seems you translated it from a file just before the changes to advance back/forwards by 10/100? 
Or was it the one based on the release `PicView 2.3.2`?

I have added the lines:
```
"AdvanceBy10Images": "Gå framåt 10 bilder", 
"GoBackBy10Images": "Gå tillbaka 10 bilder",
"AdvanceBy100Images": "Gå framåt 100 bilder", 
"GoBackBy100Images": "Gå tillbaka 100 bilder"
```
https://github.com/Ruben2776/PicView/blob/9b5e5b2dbc9674251e335ead0c125c27bca9c984/src/PicView.Core/Config/Languages/sv.json

**Comment by sparmark** [comment]
(Yes, it was based on 2.3.2. I wasn't aware that you were so effecient that you added the new texts too. 😂😂😂)
Correction: I am rather sure I copied en.json from the repository right before you added the changes +/-10 and +/- 100 images.

If it is not too much trouble, I think a better (and simpler translation) would be:

"AdvanceBy10Images": "Framåt 10 bilder", 
"GoBackBy10Images": "Bakåt 10 bilder",
"AdvanceBy100Images": "Framåt 100 bilder", 
"GoBackBy100Images": "Bakåt 100 bilder"

**Comment by sparmark** [comment]
Thank you very much for fast implementation!
